src/markdown-pages/build-apps/build-hello-world-app.mdx

```diff
@@ -222,15 +222,15 @@ nr1 create
 
   <Step>
 
-Select **catalog**, which creates a stub in your project under the `catalog` directory. Here's how the results might look in your code editor:
+Select **catalog**, which creates a stub in your project under the `catalog` directory in the root directory and any launchers, nerdlets, or visualizations directories. Here's how the results might look in your code editor:
```

Comment:
> Here's how the results might look in your code editor:

Do you think it'd be valuable to update the following screenshot?

Reply:
Ya thats probably a good idea. Let me see what I can whip up.

Reply:
Fixed in 0c7017d

Reply:
```suggestion
Select **catalog**, which creates a stub in your project under the `catalog` directory in the root directory and any launcher, Nerdlet, or visualization directories. Here's how the results might look in your code editor:
```

Reply:
@tyreer I can see how this reads better in English, but I'm wondering if wrapping it as inline code would make sense. The reason I went with plural here was because the directories are called `nerdlets`, `launchers` and `visualizations`. I guess the question here is whether we want to use the directory names or update to your suggestion. Thoughts? @alexronquillo do you have a preferred direction for this kind of thing?

Reply:
Oh also, are nerdlets capitalized or no? I know Nerdpacks are, but wasn't sure if that also applied to Nerdlet.

Reply:
Happy to follow Alex's lead here :) 

Yeah, `Nerdlet` in a sentence should be capitalized. Checked here: https://newrelic.slack.com/archives/G01GR8SHEHW/p1626718298004000?thread_ts=1626717697.002200&cid=G01GR8SHEHW

Referring to a directory, I think something like `nerdlets` would make it clear if we go that route.

Reply:
We've been trying to move away from using code for filenames because it can be confusing at times, so I think if we want to use directory names, let's use:

> ...under the _catalog_ directory in the root directory and any _launchers_, _nerdlets_, or _visualizations_  subdirectories....

I do prefer what @tyreer suggested, though, for two reasons:

1. It reads better in English
2. The new _catalog_ directories actually go into the subdirectories of those, right? So not in _launchers_, but actually in _launchers/my-laucher_

And, yep, Nerd* terms are all capitalized because they're branded terms, whereas the others are common nouns.

Reply:
> The new catalog directories actually go into the subdirectories of those, right? So not in launchers, but actually in launchers/my-laucher

Thats correct. It would be something like `launchers/my-launcher/catalog`

Reply:
Updated in 3dbe9d8